src/websocket/index.ts

```diff
@@ -75,7 +78,7 @@ export const onConnection = async (socket: WebSocket.WebSocket): Promise<void> =
     }
 
     const method_name = request.method as string
```

Comment:
I would suggest here to check that method_name is in Object.keys(eth_method). It will filter out ONLY user defined props. As an extra layer of defence